### DISCUSSION TITLE
domain transfer problem

### DISCUSSION THREAD

**Discussion by SebaAguiar** [vercel/community#6400] [Account Issue]
### Question

Recently, I purchased a domain through my Vercel account and now I need to transfer it to a different account. However, when attempting to do so, I encounter an error message stating: "The entered domain is not transferable (reason: 'Domain already exists in Vercel Inc’s account')."

I understand there may be restrictions or specific procedures for transferring domains between accounts, and I would like to know how to resolve this issue. Could you provide me with the steps to follow or assist me in the process to ensure the transfer can be successfully completed?

**Comment by amyegan** [comment]
Hey @SebaAguiar. Can you walk me through the steps you've tried so far?
